core/storage/statistics/gae_models.py

```diff
@@ -588,6 +759,15 @@ def _insert_submitted_answers_unsafe(
         """See the insert_submitted_answers for general documentation of what
         this method does. It's only safe to call this method from within a
         transaction.
+
+        Args:
+            exploration_id: str. ID of the exploration currently being played.
+            exploration_version: int. Version of exploration.
+            state_name: str. Name of current state.
+            interaction_id: str. It tells which type of computations are to be
```

Comment:
Your doc string is not right. Just use: "ID of the interaction".

Reply:
done.